dali/ft/daftformat.cpp

```diff
@@ -1065,11 +1065,10 @@ void CCsvQuickPartitioner::findSplitPoint(offset_t splitOffset, PartitionCursor
     else
     {
         // We are in the first part of the file
-        bool eof;
-        if (format.maxRecordSize + maxElementLength > blockSize)
-            eof = !ensureBuffered(blockSize);
-        else
-            eof = !ensureBuffered(format.maxRecordSize + maxElementLength);
```

Comment:
The above condition looks backward, but correct if flipped ?
i.e. if > blockSize should be using format.maxRecordSize + maxElementLength

Reply:
also, the same code is present in a few other places, if this is wrong, so are other places I think.